Content.Server/Chat/Managers/ChatSanitizationManager.cs

```diff
@@ -59,6 +59,8 @@ public class ChatSanitizationManager : IChatSanitizationManager
         { "^^/", "chatsan-waves" },
         { ":/", "chatsan-uncertain" },
         { ":\\", "chatsan-uncertain" },
+        { "lmao", "chatsan-smiles" },
+        { "lol", "chatsan-smiles" },
```

Comment:
I think `chatsan-laughs` would work better here

Reply:
Yeah actually, changing this now. that was an oversight on my part

Reply:
> I think `chatsan-laughs` would work better here

updated